CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ librdkafka v1.9.0 is a feature release:
  * Added KIP-140 Admin API ACL support (by @emasab, #2676)
 
 
+## Upgrade considerations
+
+ * Consumer:
+   `rd_kafka_offsets_store()` (et.al) will now return an error for any
+   partition that is not currently assigned (through `rd_kafka_*assign()`).
```

Comment:
does the following issue comment indicate there may additionally be a different issue with similar effect?
https://github.com/confluentinc/confluent-kafka-dotnet/issues/1672#issuecomment-1000346590

Reply:
Yep, that does look like the same issue.
A background rebalance is taking place between poll() and offsets_store(), effectively overwriting the reset offset with a proper one, which is then committed.